Транслитерација value атрибута input елемента

miljanuscumlic opened the pull request:
Пример странице на којој се користи овај атрибут - при дну (+ Učitaj još):
https://nova.rs/it/

miljanuscumlic committed:
Transliterate value attribute of an input element

turanjanin commented:
Value атрибут је проблематичан, зато га нисам иницијално убацио у логику за пресловљавање. 

Нама, што се пресловљавања интерфејса тиче, игра улогу само ако је реч о `<input type="submit">` или `<input type="button">` елементу. У супротном, вршило би се пребацивање и `<input type="text">` поља у формуларима што није жељено понашање - корисници можда желе да на неким сајтовима попуњавају податке на латиници.

Чак и код ових submit и button елемената, треба водити рачуна да ли они имају name атрибут. У случају да такав атрибут постоји, value ће се послати на сервер и потенцијално користити за неку даљу обраду, зато такве елементе треба оставити у оригиналу.

miljanuscumlic committed:
Transliterate value attribute under the precise conditions

Value attribute is transliterated only when it belongs to an input element that has no name attribute and has an type attribute with the value of button or submit.

miljanuscumlic commented:
Јоване, хвала ти на детаљном објашњењу.
Нисам имао потпуну слику пре тога.
На основу овога што си написао допунио сам услов под којим се врши транслитерација атрибута value. Ако мислиш да је ово ОК и да су ови услови довољни да се заштитимо од нежељених ситуација, онда супер.
Ако не, можемо да додатно појачамо услов или да одбацимо issue.

turanjanin commented:
Делује ми да ће ово сада радити како треба, хвала на труду :)
